x-pack/plugins/lens/public/editor_frame_service/embeddable/embeddable_factory.ts

```diff
@@ -33,6 +36,7 @@ interface StartServices {
   savedObjectsClient: SavedObjectsClientContract;
   expressionRenderer: ReactExpressionRendererType;
   indexPatternService: IndexPatternsContract;
+  dashboard?: DashboardStart;
```

Comment:
Why coupling to dashboard?

Reply:
This is temporary - see https://github.com/elastic/kibana/issues/77505